celery/backends/base.py

```diff
@@ -678,5 +686,9 @@ def _is_disabled(self, *args, **kwargs):
         raise NotImplementedError(
             'No result backend configured.  '
             'Please see the documentation for more information.')
+
+    def as_uri(self, *args, **kwargs):
+        return 'disabled'
```

Comment:
I kept the [previous behaviour](https://github.com/celery/celery/pull/3079/files#diff-5057f62807f26f9a701a0d9bf9a2a2b3L209) but makes more sense yes